.envrc

```diff
@@ -0,0 +1 @@
+export VULKAN_SDK_PATH=/Users/joe/Projects/sdks/vulkan/macOS
```

Comment:
This should not be in the PR.

Reply:
Is this used by a specific IDE? Might be worth adding to `.gitignore`.

Reply:
lol

Reply:
added to my personal gitignore, which is where it should have been from the beginning.